packages/application-components/src/components/dialogs/confirmation-dialog/confirmation-dialog.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { MessageDescriptor } from 'react-intl';
+import { commonMessages } from '@commercetools-frontend/i18n';
```

Comment:
Sorry nit: I'd call it `sharedMessages`. Here and everywhere else (readme) as that's what the package is called too.